packages/react-core/src/demos/examples/Masthead/MastheadWithUtilitiesAndUserDropdownMenu.tsx

```diff
@@ -143,14 +144,28 @@ export const MastheadWithUtilitiesAndUserDropdownMenu: React.FunctionComponent =
     </DropdownGroup>
   ];
 
-  const appLauncherItems = [
-    <ApplicationLauncherItem key="application_1a" href="#">
-      Application 1 (anchor link)
-    </ApplicationLauncherItem>,
-    <ApplicationLauncherItem key="application_2a" component="button" onClick={() => alert('Clicked item 2')}>
-      Application 2 (button with onClick)
-    </ApplicationLauncherItem>
-  ];
+  const appLauncherMenu = (
```

Comment:
This is the one we want to keep. We can update it to match the composable menu application launcher demo.